projects/kit/components/input-tag/input-tag.component.ts

```diff
@@ -295,12 +295,8 @@ export class TuiInputTagComponent
         return this.hasCleaner || this.hasTooltip || this.iconAlignRight;
     }
 
-    /**
-     * @deprecated: use `status$ | async` instead
-     * TODO: remove in v3.0
-     */
     get status(): TuiStatusT {
-        return this.modeDirective && this.modeDirective.mode ? 'default' : this.tagStatus;
+        return this.modeDirective?.mode ? 'default' : this.tagStatus;
```

Comment:
we have 2 variables about status in different parts of file.

Can we place this logic inside stream `status$` (and drop this getter) ?
Something like this
```ts
status$: Observable<TuiStatusT> = this.mode$.pipe(
        map(mode => (mode ? 'default' : this.tagStatus)),
        map(status => status || this.modeDirective?.mode ? 'default' : this.tagStatus)
    );
```

Reply:
it's probably async pipe is never emitted and return null

Reply:
I changed to 

```
status$: Observable<TuiStatusT> = this.mode$.pipe(map(() => this.status));
```